### PR TITLE
docs: add new events and remove eventemitter3 references

### PR DESCRIPTION
**Pull request by dated**
## Proposed changes

Document changes to the event-emitter, https://github.com/ArkEcosystem/core/pull/2321 & https://github.com/ArkEcosystem/core/pull/2329.

## Types of changes

- [x] Docs (documentation only changes)

## Checklist

- [x] I have read the [CONTRIBUTING](/guidebook/contribution-guidelines/contributing.html) documentation
- [x] I have read the [WRITING DOCUMENTATION](/guidebook/contribution-guidelines/writing-documentation.html) guidelines

### CODE REVIEW

**Comment by ghost** [comment]
**Thanks for submitting this pull request!** A maintainer will review this in the next few days and explicitly select labels so you know what's going on.

_If no reviewer appears after a week, a reminder will be sent out._

File: docs/guidebook/core/plugins/required/core-event-emitter.md [hunk 1]
```diff
@@ -54,6 +52,9 @@ public getEvents() {
         "transaction.applied",
         "transaction.expired",
         "transaction.forged",
+        "transaction.pool.added",
+        "transaction.pool.not-added",
```
Comment (faustbrian) [inline docs/guidebook/core/plugins/required/core-event-emitter.md]: This changed to `transaction.pool.rejected` before it was merged.

**Review by faustbrian — APPROVED** [review]

**Comment by ghost** [comment]
A contributor has approved this PR. A maintainer will merge this PR shortly. If it shouldn't be merged yet, please leave a comment saying so and we'll wait.

_Thank you for your contribution!_

**Comment by ghost** [comment]
Your pull request has been merged and marked as tier 5. _It will earn you $5 USD._

**Pull request merged**